HRC_TXING_CHECK/HRC_Txing_Check.py

```diff
@@ -18,8 +34,12 @@
 
 import ORP_File_Class as ofc
 
+
+from IPython import embed
```

Comment:
Remove unnecessary import after testing is finished

Reply:
removed